docs/release-notes/release-notes.md

```diff
@@ -15,7 +15,7 @@ if using linux in docker, kubernetes, a vm or baremetal, this can be done by cal
 make enable-bbr
 ```
 
-command on the host machine.
+Command on the host machine.
```

Comment:
seems fine to leave uncapitalized b/c I think it's a continuation of "this can be done by calling ___ command on the host machine"

```suggestion
command on the host machine.
```